src/lib/components/product_variants/ProductVariants.js

```diff
@@ -49,14 +49,19 @@ export default class ProductVariants extends Component {
       );
     }, this);
 
-    return (
-      <div className={classes}>
-        <h3 className={`${CLASS_ROOT}__h`}>
-          This product is also available as:
-        </h3>
-        <div className={`${CLASS_ROOT}__thumbs`}>{variants}</div>
-      </div>
-    );
+    if (variants.length > 0) {
```

Comment:
Looks like this is the same fix as in https://github.com/BustedTees/teepublic-react/pull/50, correct? I this the ternary is a little cleaner

Reply:
@jeffcooper86 looks like it, didn't realize there was another PR. I'll merge that one and update this one.

Reply:
I see, I'm not a huge fan of multiline ternary operators... but that's me coming from Ruby. I'll defer to you on this one. 👍 :)

Reply:
Yeah ruby is a bit different with its implicit returns